test/MessageMock.ts

```diff
@@ -90,6 +90,22 @@ export class MessageMock {
                 };
             });
 
+        nock(baseURL)
+            .persist()
+            .get("/conversation")
+            .query(function(queryObject: any) {
+                if (MessageMock.onRequestCallback) {
+                    MessageMock.onRequestCallback();
+                }
+                return queryObject.uuid === "delay";
+            })
+            .delayConnection(10000)
```

Comment:
once the environment variables are in place, this can be set to a smaller value (setting the timeouts to 2,5 and keeping this at 10

Reply:
this is just a test

Reply:
Yes, but the nock will await for 10 seconds, instead using the environment variables now present on the code you can reduce the retry to 2 milliseconds and have a small delay of 10 milliseconds